Cargo.toml

```diff
@@ -88,14 +96,14 @@ dashmap = "6.1.0"
 divan = "0.1.21"
 env_logger = "0.11.8"
 either = "1.15.0"
-ethers = { version = "2.0.14", default-features = false } # forced dependency 'ring' ignored in audit
```

Comment:
Pls do `cargo update` and remove the `ring` line in the `.cargo/audit.toml` exceptions list, as well as all the other ethers.rs stuff, that is exceptioned, and if needed, please, introduce new exception lines for alloy based on the audit.